editor.planx.uk/src/@planx/components/Pay/Public/InviteToPayForm.tsx

```diff
@@ -220,6 +232,13 @@ const InviteToPayForm: React.FC<InviteToPayFormProps> = ({
             }}
           />
         </InputLabel>
+        <StyledWarningBox>
+          <ErrorOutline />
+          <Typography variant="body2" ml={2} fontWeight="bold">
+            Selecting "continue" locks your application and you'll no longer be
+            able to make changes.
```

Comment:
any strong opinions if this text should be editor-configurable? it's quite general and vague (eg no expiration date, etc) so i figured for now it's not likely to change?

Reply:
No I would say that letting editors configure this is not necessary 👍